apps/www/package.json

```
@@ -26,7 +26,7 @@
     "@radix-ui/react-checkbox": "^1.0.4",
     "@radix-ui/react-collapsible": "^1.0.3",
     "@radix-ui/react-context-menu": "^2.1.4",
-    "@radix-ui/react-dialog": "^1.0.4",
+    "@radix-ui/react-dialog": "^1.0.5",
```

```suggestion
    "@radix-ui/react-dialog": "^1.0.4",
```

Going to leave this version alone since this should still install the `1.0.5` version with the `^` specificier.

---

This should be good to run the checks again @shadcn